src/index.js

```diff
@@ -239,11 +239,13 @@ class Dropzone extends React.Component {
     // execute the timeout only if the FileDialog is opened in the browser
     if (this.isFileDialogActive) {
       setTimeout(() => {
-        // Returns an object as FileList
-        const { files } = this.fileInputEl
+        if (this.fileInputEl != null) {
+          // Returns an object as FileList
+          const { files } = this.fileInputEl
 
-        if (!files.length) {
-          this.isFileDialogActive = false
+          if (!files.length) {
```

Comment:
simpler to add `if (!files || !files.length)` add remove this `if (this.fileInputEl != null) {`

Reply:
I don't know. The PR code looks more readable to me

Reply:
for me more readable this version 
```
if (this.isFileDialogActive) {
                setTimeout(() => {
                    const { files } = this.fileInputEl;

                    if (!files || !files.length) {
                        this.isFileDialogActive = false;
                    }
                    if (typeof onFileDialogCancel === 'function') {
                        onFileDialogCancel();
                    }
                }, 300);
            }
```

Reply:
no need one more if and what about case with `files undefined`